README.md

```diff
@@ -4,6 +4,19 @@
 
 It can also be used as a Substrate provider to query or to interact with the Reef chain using the same calls as in the [Polkadot.js](https://polkadot.js.org/docs/api).
 
+## Installation
+
+Install dependencies with `yarn` [see issue](https://github.com/reef-defi/evm-provider.js/issues/5#issuecomment-912389541).
+
+### Yarn
+
+```bash
+yarn add @reef-defi/api
+yarn add @reef-defi/api-derive
+yarn add @reef-defi/types
+yarn add @reef-defi/type-definitions
```

Comment:
Those 4 packages are not necessary. I would only leave the line below.

Reply:
I see, the example is outdated. I took the liberty to update the `README` with the newer version, omitting `@reef-defi/api*` references. See [PR](https://github.com/reef-defi/evm-provider.js/pull/8). Please rebase this on top of it - without the `@reef-defi/api*` so I do not rob you of your contribution.

Reply:
I noticed the comment about yarn on the bottom was still there in your changes, but I've moved that comment under Installation.

Reply:
Thank you.